### PR TITLE
improved styling

### LINKED ISSUE #56: Improve styling of classes, enums, etc. [3]
This is how a class currently looks like:
image

This is what classes in the Eclipse RCP editor look like:
image

I suggest to improve the styling to make Ecore-GLSP look more appealing. It doesn't have to be perfectly aligned to how it looks like in Eclipse RCP, but we should try to make Eclipse-GLSP look at least as good as Eclipse RCP (hopefully even better and more modern) :).
In particular, I see the following aspects that should be improved (feel free to open separate issues for those items or add items):
- [ ] Icons instead of the circles with letters (ideally these icons are SVGs, but we can also use the PNGs from Ecore)
- [ ] A border between the class title and the attributes
- [ ] More modern looking background color (see also #51)
- [ ] Less padding
- [ ] Less border-radius
- [ ] Smaller font size

### PR DESCRIPTION
**Pull request by leodennis**
fixes #56 

The only issue is the change to 
client\sprotty-ecore\src\feedback.tsx
I tried fix it so that it looks good but I have no Idea how to write it better...

### CODE REVIEW

File: client/sprotty-ecore/css/diagram.css [hunk 1]
```diff
@@ -51,7 +52,7 @@ svg {
 }
 
 .sprotty-label-node>.sprotty-label {
-    alignment-baseline: middle;
+    alignment-baseline: mi;
```
Comment (CamilleLetavernier) [inline client/sprotty-ecore/css/diagram.css]: "mi" is not a valid value; I don't really get this change

File: client/sprotty-ecore/src/feedback.tsx [hunk 2]
```diff
@@ -29,13 +29,25 @@ const JSX = { createElement: snabbdom.svg };
...
+            }
+          }
+        });
+      }
+
       const feedback: VNode = (
         <rect
           x={-hPadding}
-          y={-element.bounds.height / 2 - vPadding}
+          y={-element.bounds.height / 2 - vPadding + offsetY/2-1}
```
Comment (CamilleLetavernier) [inline client/sprotty-ecore/src/feedback.tsx]: Magic numbers like `-1` should be avoided

**Review by CamilleLetavernier — CHANGES_REQUESTED** [review]
It looks much better! :)

Regarding the label feedback, can you clarify what you mean? Is it an issue about the way it looks, or the way it is implemented?

If it's about the look, here are a few examples of how label selection work in Ecore Tools:

image

image

image

(For the first image, the orange is the default selection color on Linux; it would be blue on Windows)

So maybe we could use a label background color instead of a rectangle overlay; but then we may have to integrate that directly in the NodeLabelView rather than using a separate feedback element (Inserting a rectangle below the existing label might be an issue)

Also, I think we should reduce the overall size of the elements (Font size and SVG size). We can always zoom out, but having a smaller default size would be best

**Comment by leodennis** [comment]
@CamilleLetavernier thanks for the input, I fixed the issues and found a simpler version for the feedback. The styling of the selection could be changed as well if wanted but maybe in a different issue.

**Review by CamilleLetavernier — APPROVED** [review]
The changes look good to me. I just noticed that the label position is off on Firefox, but that was already the case before the patch:

image

image

**Pull request merged**